intellij-plugin-structure/structure-intellij/src/main/resources/com/jetbrains/plugin/structure/intellij/problems/plugin-problems.json

```diff
@@ -2,7 +2,8 @@
   "existing-plugin": {
     "com.jetbrains.plugin.structure.intellij.problems.ForbiddenPluginIdPrefix": "ignore",
     "com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginId": "ignore",
-    "com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginName": "ignore"
+    "com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginName": "ignore",
+    "com.jetbrains.plugin.structure.intellij.problems.InvalidUntilBuildWithJustBranch": "warning"
```

Comment:
For which plugins will it be error then? Maybe the level should be `unacceptable warning`, so for all already uploaded plugins it will be considered on PV side as a warning and for all new uploading versions it will be considered as an error on MP side...

Reply:
`InvalidUntilBuildWithJustBranch` is
-  _error_ for new plugins - as per Kotlin code
- _warning_ for existing plugins - as per remapping in JSON

Reply:
There is a _warning_ for new plugins in JSON.

Reply:
Synchronized with JetBrains plugins.